resources/assets/js/stories/FindSkillsDialog.stories.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import { withIntl } from "storybook-addon-intl";
+import { action } from "@storybook/addon-actions";
+import { useState } from "@storybook/addons";
+import { fakeSkill2, fakeSkill4, fakeSkills } from "../fakeData/fakeSkills";
+import { fakeSkillCategories } from "../fakeData/fakeSkillCategories";
+import FindSkillsDialog, {
+  FindSkillsDialogTrigger,
+} from "../components/FindSkillsDialog/FindSkillsDialog";
+
+const stories = storiesOf("Components/Find Skills Dialog", module).addDecorator(
```

Comment:
Can we move this into "Applicant Profile/Skills"? I always forget where this Story is in the menu 😅